dkandalintsev/login.feature

```diff
@@ -0,0 +1,127 @@
+Feature: 1. Signing in with Google account
+
+Scenario: 1.01 Sign in page is available for user
+    Given user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then https://accounts.google.com/ authorization page opens
+
+Scenario: 1.02 User has one saved Google account on device and tries to sign in
+    Given user is signed in with one Google account on google.com in the web browser
```

Comment:
The good main idea for the test and good using tables in the scenario! 
 But, please, review how we can make better the "Given" section. 

```
Given user signed to Scoring portal with following credentials:
        | email address               | phone number | password |
        | testkandalintsev@gmail.com  | +79819139630 | Test123! |
And user signed out
.... 
```
This improvement can be sued for the similar cases below.